eng/tools/BaselineGenerator/Program.cs

```diff
@@ -26,34 +32,57 @@ static void Main(string[] args)
 
         private readonly CommandOption _source;
         private readonly CommandOption _output;
+        private readonly CommandOption _update;
 
         public Program()
         {
-            _source = Option("-s|--source <SOURCE>", "The NuGet v2 source of the package to fetch", CommandOptionType.SingleValue);
+            _source = Option("-s|--packageSource <SOURCE>", "The NuGet source of packages to fetch", CommandOptionType.SingleValue);
             _output = Option("-o|--output <OUT>", "The generated file output path", CommandOptionType.SingleValue);
+            _update = Option("-u|--update", "Regenerate the input (Baseline.xml) file.", CommandOptionType.NoValue);
 
             Invoke = () => Run().GetAwaiter().GetResult();
         }
 
         private async Task<int> Run()
         {
             var source = _source.HasValue()
-                ? _source.Value()
-                : "https://www.nuget.org/api/v2/package";
```

Comment:
- This value wasn't an actual package source anyhow
- Should we use the V3 feed by default?

Reply:
> Should we use the V3 feed by default?

Sure, if it works.